packages/react-ui-components/src/CheckBox/checkBox.spec.tsx

```diff
@@ -25,13 +28,6 @@ describe('<CheckBox/>', () => {
         expect(wrapper.prop('className')).toContain('fooClassName');
     });
 
-    it('should allow the propagation of additional props to the wrapper.', () => {
```

Comment:
`foo` is obviously not a valid prop for `CheckBox`. 
Is this test just wrong or is this actually a wanted functionality?

Reply:
see my comment on Headline